modules/tasks/pom.xml

```diff
@@ -137,6 +137,11 @@
             <artifactId>powermock-module-junit4</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
can not get synapse core as a dependency, task is building before the core

Reply:
(+1) Mocked the TaskManager interface instead of the quartz implementation.